tests/unit/executor/test_files.py

```diff
@@ -8,9 +8,88 @@
 import pandas as pd
 
 from tests.unit.executor_test_base import BaseExecutorDummyML
+from mindsdb.utilities.exception import QueryError
 
 
 class TestFiles(BaseExecutorDummyML):
+
+    @pytest.fixture(autouse=True)
+    def setup_and_teardown(self):
+        """Setup and teardown for each test"""
+        yield
+        try:
+            for file_name in list(self.file_controller.list_files()):
+                try:
+                    self.file_controller.delete_file(file_name)
+                except Exception:
+                    pass
+        except Exception:
+            pass
+
+    @pytest.fixture(autouse=True)
+    def create_tables_and_data(self):
```

Comment:
these data are not used by every test, it is more test specific data
maybe is better to create separated class (with all tests that use these files) and create these files once in `setup_class`?

Reply:
Thanks @ea-rus i will take a look now